### ISSUE TITLE
VT: Switch to using official API

### ISSUE THREAD

**Issue by mileswwatkins** [openstates/openstates#2040]
Vermont's got an official API now, which we should look into leveraging it. Requesting a key to check it out.

https://legislature.vermont.gov/docs/api/v1/index.html

**Comment by benjamin-brown** [comment]
That's very interesting! Thanks for the heads up, @mileswwatkins !

**Comment by mileswwatkins** [comment]
(Currently supports only bill endpoints.)

**Comment by In-vincible** [comment]
@mileswwatkins should I try writing one bill scraper or whatever we want to call it, via APIs. For getting the API key I couldn't find direct registration process, I think You have to send mail to IT@leg.state.vt.us.

**Comment by mileswwatkins** [comment]
Yup, if you want to replace the `vt/bills.py` module with code that points at the new API, go for it! You may want to look at other states that use REST APIs, to see how they structure their scraping code, or instantiate an API client locally.

**Comment by mileswwatkins** [comment]
I looked into this a bit; from what I can tell, the API isn't _quite_ ready for us to use. I'm reaching out to the developers to see if I'm mistaken, and see if there's a public development roadmap that they could share.

One example issue is that their `/bill/list` endpoint _only_ lists bills, _not_ resolutions, so we wouldn't even be able to switch over to that entirely for our `Bill` scraping. This is odd since their bill-detail endpoint _can_ return information on resolutions. I tried poking at `/resolution/list` to see if it existed without documentation, but no luck.

**Comment by jamesturk** [comment]
following up with the developers as well, to see if anything new came about since 2018

**Comment by benjamin-brown** [comment]
I've been checking in periodically, don't see any new changes with VT's API page
https://legislature.vermont.gov/docs/api/v1/index.html#

Haven't spoken to developers though, so not sure if this is all they have, or if maybe there's more that just isn't publicly available yet...

**Comment by jamesturk** [comment]
I tried to reach out to them, no response yet.  Marking this issue as upstream though since it isn't ready for primetime as-is.
